tests/runtime/json-output

```diff
@@ -190,3 +190,14 @@ NAME print bool type
 RUN {{BPFTRACE}} -q -f json -e 'BEGIN { print((true, false)); exit(); }'
 EXPECT {"type": "value", "data": [true,false]}
 TIMEOUT 1
+
+NAME BENCH
+RUN {{BPFTRACE}} -q -f json -e 'BENCH:a { @ = count(); } BEGIN { print(@); clear(@); exit(); }'
```

Comment:
why do we need the `BEGIN` probe?

Reply:
Shouldn't you still run `BEGIN` first? My assumption would be that `BEGIN` could still be used to set up some common state.

Reply:
> My assumption would be that BEGIN could still be used to set up some common state.

Yeah this is a good point. `END` can also be used to clear maps you don't want printed to the console.

Reply:
Makes sense to me. I'll make it so `BEGIN` and `END` always run first and last, even in benchmark mode.